docs/RISKS.md

```diff
@@ -0,0 +1,40 @@
+| 3 | Misunderstandment of requirements | Requirements are the foundation of any project, and if they are misunderstood, it can result in a project that does not meet the intended goals and objectives. This can lead to wasted time, effort, and resources, as well as frustration among team members and the customer (Kay). | 0,3 | 5 | 1,5 | Before a ticket is put into a sprint, it is talked about in the team. Acceptance criteria are discussed to make sure the key points are clear to everyone. The team is obligated to fulfill the acceptance criteria only. | Jens, Markus, Jonas, Chris, Lars |
+| 4 | Unknown complexity when creating tickets | The scope of the project may expand beyond what the team can handle, leading to delays and possible failure. | 0,2 | 7 | 1,4 | To mitigate this risk, tickets are prioritized and broken down into smaller parts. Regular reviews deepens the understanding on complexity and technical ways to implement the features, helping the project manager to get a feeling for the involved complexity for a new ticket. | Jens |
+| 5 | Technology failures | This may include breaking changes in our tech stack requiring immediate changes, as well as unexpected behavior of technology, ultimately leading to the result that the goal of the project is not achievable. | 0,1 | 10 | 1 | This risk can be best mitigated by two strategies: (a) choosing reliable technology with a record of stability as well as reliability (following best practices), (b) training team member to employ them with the power to response to technology failure immediately on their own behalf | Jonas |
+| 6 | Code quality / Technical debts | Technical debt refers to the shortcuts or compromises taken in the development process that can lead to problems down the road. This can include poorly written code, incomplete documentation, lack of testing, or outdated technology stacks. | 0,2 | 3 | 0,6 | This risk can be mitigated by establishing coding standards and best practices. By conducting code reviews, focus is taken on the possibility for further extension and growth. The team is reminded of the wish for refactoring whenever helpful, to adress technical debts early on and get rid of those. Tests can be used as a measure of confidence, so refactoring can be done easier because the behavior is tested. This includes unit tests, integration tests as well as regression tests. The team is aware that taking shortcuts is not wanted and will hinder us down the path when it'll become the most stressful. | Markus |
+| 7 | Hardware problems | Hardware may fail or malfunction for any reason, including theft. Firmware or human failures may result in a lock-out of our developers so that the hardware becomes unusable for our project. | 0,2 | 9 | 1,8 | This risk can be mitigated by regular maintenance and testing. By making sure all important work is pushed to GitHub, we have another layer of security in case someone looses access to their laptop. | Jens |
```

Comment:
```suggestion
| 7 | Hardware problems | Hardware may fail or malfunction for any reason, including theft. Firmware or human failures may result in a lock-out of our developers so that the hardware becomes unusable for our project. | 0,1 | 9 | 0,9 | This risk can be mitigated by regular maintenance and testing. By making sure all important work is pushed to GitHub, we have another layer of security in case someone looses access to their laptop. | Jens |
```

I dont think the probability of being locked out of our hardware is as high as getting ill